deepspeed/module_inject/test_module_parser.sh

```diff
@@ -0,0 +1,9 @@
+#Runs through all huggingface models and runs module_parser
+
+set -x
+
+model_dir="./transformers/src/transformers/models"
+
+for d in $model_dir/*/modeling_*.py; do
+	python module_parser.py -f $d
```

Comment:
Would this style test be a good candidate to add into our unit tests? Might be a good way to detect when `transformers` makes changes that break our assumptions?

Reply:
In a next phase, I'm going to work on changing the parser such that we parse the model structure from the pipeline instead of the transformers repo. That way we don't need to store injection policies and instead generate them at runtime. And we shouldn't need to check for changes to the transformer repo.